test/index.spec.js

```diff
@@ -2,53 +2,92 @@
 'use strict'
 
 const expect = require('chai').expect
-const mh = require('multihashes')
+const multihash = require('multihashes')
 const multihashing = require('multihashing')
```

Comment:
Definitely something happening next week, however, it would be impossible for me to focus on the IPLD work at hand, if adding to the npm link mad mountain that I had to do, I also had to maintain branches of branches npm linking all over the place :)